configs/jsactions/rollup.config.js

```diff
@@ -0,0 +1,119 @@
+import { red, yellow } from "colors";
+import { mkdirSync, promises } from "fs";
+import { join, relative } from "path";
+import clear from "rollup-plugin-clear";
+import command from "rollup-plugin-command";
+import { cp } from "shelljs";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import typescript from "@rollup/plugin-typescript";
+import { collectDependencies } from "../../packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies";
+import { copyClientDependencies } from "./rollup-plugin-copy-client-dependencies";
+
+const cwd = process.cwd();
+
+export default async args => {
+    const jsActionTargetFolder = `javascriptsource/${args.configProject ?? "nativemobileresources"}/actions`;
+    const result = [];
+    const files = await findFiles();
+    const outDir = join(cwd, "dist");
+
+    files.forEach((file, i) => {
+        const fileInput = file.path.replace(join(cwd, "/"), "");
+        const [fileOutput] = file.name.split(".");
+        result.push({
+            input: fileInput,
+            output: {
+                format: "es",
+                file: join(outDir, `${fileOutput}.js`),
+                sourcemap: false
+            },
+            external: nativeExternal,
+            plugins: [
+                ...(i === 0 ? [clear({ targets: [outDir] })] : []),
+                collectDependencies(nativeExternal, outDir, true, fileOutput, i === 0, true),
+                ...(i === files.length - 1 ? [copyClientDependencies(clientDependencies, outDir)] : []), // Copy extra dependencies not listed in lib's package.json
+                nodeResolve({ preferBuiltins: false, mainFields: ["module", "browser", "main"] }),
+                typescript({
+                    noEmitOnError: false,
+                    sourceMap: false,
+                    inlineSources: false,
+                    target: "es2019",
+                    types: ["mendix-client", "big.js", "react-native"],
+                    allowSyntheticDefaultImports: true
+                }),
+                command([
+                    () => {
+                        if (i === files.length - 1) {
```

Comment:
you can move this "if" out of `command`, that would be more logical

Reply:
Only if you want that do be executed for each JS action

Reply:
what i mean is `i === file.length - 1 ? command([() => {...}]) : null

Reply:
Done